examples/protocol_specification_ex/sample.yaml

```diff
@@ -5,10 +5,10 @@ license: Apache 2.0
 description: 'A protocol for negotiation over a fixed set of resources involving two parties.'
 speech_acts:
   cfp:
-    query: DataModel
+    query: "DataModel"
```

Comment:
I am wondering whether it would be preferable to have a prefix for user defined types as well. Makes also the logic more easy to read. Something like, `st:DataModel`

Reply:
Yes good idea. 

Maybe "ct:BlaBla" for custom type?